test/e2e-tests.sh

```diff
@@ -29,11 +29,12 @@ if [ "${EVENTING_KAFKA_BROKER_CHANNEL_AUTH_SCENARIO:-""}" != "" ]; then
 fi
 
 if [[ -z "${BROKER_CLASS}" ]]; then
-  fail_test "Broker class is not defined. Specify it with 'BROKER_CLASS' env var."
-else
-  echo "BROKER_CLASS is set to '${BROKER_CLASS}'. Running tests for that broker class."
+  BROKER_CLASS=KafkaBroker
```

Comment:
on the actual test config, on the infra repo, we do populate them. 

Perhaps it's also good to reflect this in the `DEVELOPMENT` md file? 

/cc @aliok

Reply:
I am ok with it. I've seen this change suggested many times now :)